company_website/static/main_page/menu.sass

```diff
@@ -0,0 +1,230 @@
+@import "partials/set_initials"
+@import "partials/constants"
+@import "partials/hamburger_menu"
+
+.hidden
```

Comment:
Same here about class wrapper.

Reply:
I think they are wrapped now. Rest classes are on the same level or they are not nested to other classes.